app/assets/javascripts/modules/coronavirus-landing-page.js

```diff
@@ -42,5 +43,21 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     })
   }
 
+  CoronavirusLandingPage.prototype.addTimelineCountrySelector = function () {
+    var timelineRadios = document.querySelector('.js-change-location')
+    if (timelineRadios) {
```

Comment:
It would be cleaner but it would add fractionally more JS for users to download. Given it's not a complicated chunk of code I'm inclined to leave it as is.